thirdparty/playtiles.md

```diff
@@ -0,0 +1,9 @@
+## Playtiles
+
+Third party consumer of the Homebrew Hub API. Entries marking `playtiles` in their `third-party` metadata field are allowing 'Playtiles' to re-distribute/embed the entry metadata and assets in their clients under the conditions listed in this page.
+
```

Comment:
Can we add here a small description of what "Playtiles" is and mention the fact that it's a commercial, for profit product?